### PR TITLE
no meow desktop (bug 1167350)

### PR DESCRIPTION
**Pull request by ngokevin**

### CODE REVIEW

File: src/media/js/content_filter.js [hunk 1]
```diff
@@ -43,7 +47,7 @@ define('content_filter',
             return {};
         }
         return {
-            doc_type: filterContent
+            doc_type: settings.meowEnabled ? filterContent : 'webapp'
```
Comment (diox) [inline src/media/js/content_filter.js]: I would have done that above with the rest of the `filterContent` checks, but that's just me.

**Comment by diox** [comment]
r+

File: src/media/js/content_filter.js (hunk 1 above)
Comment (ngokevin) [inline src/media/js/content_filter.js]: just makes sure it's totally waffled, since the input can trigger a change

**Pull request merged**